solutions/solution-basic/src/main/resources/microservices.yaml

```diff
@@ -0,0 +1,20 @@
+# order of this configure file
+servicecomb-config-order: 100
```

Comment:
order default to -100?
because it should be override by normal customer microservice.yaml